extensions/amp-render/1.0/test/test-amp-render.js

```diff
@@ -163,5 +185,32 @@ describes.realWin(
       const text = await getRenderedData();
       expect(text).to.equal('Hello ');
     });
+
+    it('re-fetches json on refresh action', async () => {
```

Comment:
Suggestion: tests cases that refresh does nothing for the guarded conditions (`!src || isAmpStateSrc(src) || isAmpScriptSrc(src)`)

Reply:
Done.